website/docs/src/content/index.mdx

```diff
@@ -42,6 +42,76 @@ LLM workflows are fundamentally about function composition and data flow - exact
 
 While GenSX uses JSX, it does not share the same execution model as React, and has zero dependencies.
 
+## GenSX Cloud
+
+[GenSX Cloud](/docs/cloud) provides everything you need to ship production grade agents and workflow including a serverless runtime designed for long-running workloads, cloud storage to build stateful workflows and agents, tracing and observability, and features for team collaboration.
+
+### Serverless deployments
+
+Deploy any workflow to a hosted API endpoint with a single command. The GenSX cloud platform handles scaling, infrastructure management, and API generation automatically:
+
+```bash
+# Deploy your workflow to GenSX Cloud
+$ npx gensx deploy ./src/workflows.tsx
```

Comment:
We should separate this into a separate code block. Also worth being consistent about if we use `npx` in each of the examples or not (i see the one below doesn't have it)